backends/blocked/ceed-blocked-operator.c

```diff
@@ -384,39 +445,37 @@ static int CeedOperatorApply_Blocked(CeedOperator op, CeedVector invec,
       case CEED_EVAL_CURL:
         break; // Not implimented
       }
-    }
-  }
+      // Restrict active output block
+      if (i == activeout) {
+        ierr = CeedOperatorFieldGetLMode(opoutputfields[i], &lmode);
```

Comment:
I am currently only doing this for the active output, but I can (and probably should) switch to all outputs

Reply:
Updated